pymc/gp/hsgp_approx.py

```diff
@@ -385,14 +469,14 @@ def _build_conditional(self, Xnew):
 
         Xnew, _ = self.cov_func._slice(Xnew)
 
-        eigvals = calc_eigenvalues(self.L, self._m, tl=pt)
-        phi = calc_eigenvectors(Xnew - X_mean, self.L, eigvals, self._m, tl=pt)
+        eigvals = calc_eigenvalues(self.L, self._m)
+        phi = calc_eigenvectors(Xnew - X_mean, self.L, eigvals, self._m)
         i = int(self._drop_first is True)
 
-        if self._parameterization == "noncentered":
+        if self._parametrization == "noncentered":
```

Comment:
FWIW, either spelling is correct.

Reply:
Is it?? Damn, all these changes for nothing then 😅